Lesson30/Firat/my-app/src/components/Exercise5/index.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useState } from "react";
+
+export const Exercise5 = () => {
+  const [coordinates, setCoordinates] = useState({ x: 0, y: 0 });
```

Comment:
You don't need to change the code for this exercises but as a suggestion for the future, I'd set initial value of coordinates to 'undefined' (`const [coordinates, setCoordinates] = useState(undefined);`) as `{ x: 0, y: 0 }` may be interpreted as an actual value even though we don't have any coordinates before the first mouse click.